Working-Sessions/Threat-Model/TM-What-can-go-wrong-with-JuiceShop.md

```diff
@@ -6,7 +6,7 @@ track        : Threat Model
 technology   :
 related-to   : Juice Shop
 when-day     : Tue
-when-time    : PM-1
+when-time    : eve-3
 location     : Room-6
```

Comment:
Also we can't use the Rooms in the Evening, so we have to use of the Villas

From https://owaspsummit.org/pages/schedule/by-room/Wed.html I can see that ```Villa-2``` is available